test/e2e-wi-tests.sh

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env bash
```

Comment:
Like `chmod +x #!/usr/bin/env bash`? I have a dumb question...why we need chmod +x here, but not have it in e2e-tests.sh?

Reply:
or add the `chmod +x e2e-wi-tests.sh` in next line? I think I know why we need chmod + x here, otherwise, it does not have permission to run the file.

Reply:
`chmod +x e2e-wi-tests.sh` works

Reply:
I'm curious how this works. If the file isn't executable, then how is the `chmod` line run? And if it is executable, why is the line needed?

Reply:
I am curious too...as I checked the E2E test result, actually it would show things like `file not exist` when running this `e2e-wi-tests.sh`, and the then the file will continue to run for the next lines. But somehow, it works... (before adding it, would show error like `doesn't have permission to run the file`, and `e2e-wi-tests.sh` file cannot run). Let me remove the this command to try again. There might be other way under it.

Reply:
Looks like the file is already executable...I removed that line